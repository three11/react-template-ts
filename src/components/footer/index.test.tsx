import * as React from 'react';
import { render } from '@testing-library/react';

import { Footer } from '.';

test('Footer component should render successfully', () => {
	const { asFragment } = render(<Footer />);

	expect(asFragment()).toMatchSnapshot();
});
