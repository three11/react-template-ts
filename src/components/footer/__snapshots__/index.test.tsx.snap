// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render successfully 1`] = `
<DocumentFragment>
  <footer
    class="c-footer"
  >
    <div
      class="o-shell"
    >
      <p>
        Â© 2023. All rights reserved.
      </p>
      <p>
        Change language:
        <button>
          de
        </button>
        <button
          disabled=""
        >
          en
        </button>
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
